### PR TITLE
ConnectDataWithApiFinished

### PR DESCRIPTION
**Pull request by MartinNicolasRusso**
I`ve just finished my second task Connect data with Api.
image

### CODE REVIEW

File: src/containers/CryptoList/index.tsx [hunk 1]
```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {FlatList, SafeAreaView} from 'react-native';
 import Crypto from '../../components/Crypto';
-import ApiCryptos from '../../store/Api';
 import {Button} from './styles';
 
 const CryptoList = () => {
+  const [coins, setCoins] = useState([]);
```
Comment (tomasdematos) [inline src/containers/CryptoList/index.tsx]: missing type

File: src/containers/CryptoList/index.tsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
+  const cryptos = async () => {
+    const res = await fetch(
+      'https://data.messari.io/api/v2/assets?fields=id,name,slug,symbol,metrics/market_data',
```
Comment (tomasdematos) [inline src/containers/CryptoList/index.tsx]: the base url should be stored in a .env file

File: src/containers/CryptoList/index.tsx [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
+    );
+    const data = await res.json();
+    setCoins(data.data);
+  };
+  useEffect(() => {
+    cryptos();
```
Comment (tomasdematos) [inline src/containers/CryptoList/index.tsx]: use a more descriptive name

File: src/containers/CryptoList/index.tsx [hunk 4]
```diff
@@ -1,16 +1,27 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {FlatList, SafeAreaView} from 'react-native';
 import Crypto from '../../components/Crypto';
-import ApiCryptos from '../../store/Api';
 import {Button} from './styles';
 
 const CryptoList = () => {
+  const [coins, setCoins] = useState<string[]>([]);
```
Comment (thom-as-thom) [inline src/containers/CryptoList/index.tsx]: You should use the Crypto type

**Pull request merged**